### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by bripc**

### CODE REVIEW

File: src/app/form-builder/form-builder.component.html [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
   </div>
 
-  <div class="col-3 text-center p-3" id="element-properties">
+<!-- TODO: this one should only appear if a prompt type is selected for editing -->
+  <div class="col-3 p-3" id="element-properties">
     <app-form-builder-properties [groupId]="form.selectedGroupId">
       </app-form-builder-properties>
   </div>
+
+  <!-- TODO: these should only appear when a section is available to add to -->
```
Comment (russon77) [inline src/app/form-builder/form-builder.component.html]: please move TODOs to Github issues! that way we can't forget about them :-)

File: src/app/form-builder/form-builder-properties/form-builder-properties.component.html [hunk 2]
```diff
@@ -1,12 +1,12 @@
-<h2>Properties</h2>
+<h3>Item Properties</h3>
 <div id="element-properties-controls-container"
      *ngIf="item$ | async ; let item">
   <div class="form-group">
 
-    <fieldset>
-      <legend>Base</legend>
+    <fieldset class="mb-4">
+      <legend>Machine Readable Name</legend>
```
Comment (russon77) [inline src/app/form-builder/form-builder-properties/form-builder-properties.component.html]: by "Base" i meant the __base__ item properties, which I think we should adapt to "Base Properties"

File: src/app/form-builder/form-builder-properties/form-builder-properties.component.html [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
-      <label for="item-name">Item Name</label>
+      <label class="sr-only" for="item-name">Item Name</label>
```
Comment (russon77) [inline src/app/form-builder/form-builder-properties/form-builder-properties.component.html]: please restore the always visible label along with the change to the legend

File: src/app/form-builder/form-builder-groups/form-builder-groups.component.html [hunk 4]
```diff
@@ -19,31 +19,39 @@
...
+          <button type="button"
+                  class="btn btn-secondary select"
+                  (click)="onSelectGroup(group)">
+            <i class="fa fa-list"
+               *ngIf="groupId !== group.id"></i>
+            <i class="fa fa-check-square-o"
+               *ngIf="groupId === group.id"></i>
+          </button>
+        </li>
+      </ul>
```
Comment (russon77) [inline src/app/form-builder/form-builder-groups/form-builder-groups.component.html]: i disagree with the change here. i'd like to keep the button group, maybe move to the button toolbar, as seen here https://getbootstrap.com/docs/4.0/components/button-group/ . according to bootstrap we can manage with accessibility okay!

**Review by tristanmkernan — CHANGES_REQUESTED** [review]
looking good, the copy changes are great and the issues are minor!

**Comment by russon77** [comment]
@bripc in the interest of time i have made the appropriate changes and merged the code. nice work!

**Pull request closed without merging**